sdk/metricsadvisor/ai-metrics-advisor/recordings/browsers/aad_metricsadvisoradministrationclient_datafeed_datafeed/recording_creates_an_azure_blob_datafeed.json

```diff
@@ -20,62 +20,62 @@
     "report-to": "{\"group\":\"network-errors\",\"max_age\":86400,\"endpoints\":[{\"url\":\"https://identity.nel.measure.office.net/api/report?catId=GW+estsfd+est\"}]}",
     "strict-transport-security": "max-age=31536000; includeSubDomains",
     "x-content-type-options": "nosniff",
-    "x-ms-ests-server": "2.1.12231.7 - NCUS ProdSlices",
-    "x-ms-request-id": "66610f1e-74e4-4ee3-930f-ee368b654400"
+    "x-ms-ests-server": "2.1.12261.15 - WUS2 ProdSlices",
+    "x-ms-request-id": "3a91d79b-b8c8-4430-8e94-5427f82a2600"
    }
   },
   {
    "method": "POST",
-   "url": "https://endpoint/metricsadvisor/v1.0/dataFeeds",
+   "url": "https://endpoint//metricsadvisor/v1.0/dataFeeds",
```

Comment:
The recordings in this PR are also failing bcz of the extra `/` added. I didn't realize this earlier :( @HarshaNalluru @jeremymeng Any remedies?

Reply:
I removed the trailing / in the replacement variable url